Controllers/param.js

```diff
@@ -0,0 +1,76 @@
+const Cart = require('../Database/cart');
+const User = require('../Database/user');
+const Product = require('../Database/product');
+const Wishlist = require('../Database/wishlist');
+const Address = require('../Database/address');
+var jwt = require('jsonwebtoken');
+
+const isAuthorized = (req, res, next) => {
+	const token = req.headers.authorization;
+	try {
+		const decoded = jwt.verify(token, 'secret');
+		req.user = { userId: decoded.userId };
```

Comment:
Is there a reason that you did- `req.user = { userId: decoded.userId }`  nesting here? `req.userId = decoded.userId` would have worked.

Reply:
done